environments/se/core/src/main/java/org/jboss/weld/environment/se/events/ContainerInitialized.java

```diff
@@ -37,4 +37,8 @@ public ContainerInitialized(String containerId) {
         super(containerId);
     }
 
+    @Override
+    public String toString() {
+        return ContainerInitialized.class.getSimpleName() + " event for container with ID: " + getContainerId();
```

Comment:
`return getClass().getSimpleName() + " event for container with ID: " + getContainerId();`